boards/arm/nucleo_l476rg/nucleo_l476rg.dts

```diff
@@ -31,3 +31,15 @@
 	pinctrl-names = "default";
 	status = "ok";
 };
+
+&spi1 {
+	status = "ok";
+};
+
+&spi2 {
```

Comment:
If we enable SPI2 port here, we should also enable it in Kconfig.defconfig.

Reply:
done

Reply:
I removed the node, since the corresponding pinmux defines are missing.
See https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/10601/3/tests